info/container.go

```diff
@@ -467,6 +467,8 @@ type Usage struct {
 type DerivedStats struct {
 	// Time of generation of these stats.
 	Timestamp time.Time `json:"timestamp"`
+	// Latest instantaneous sample.
+	LatestUsage Usage `json:"latest_usage"`
```

Comment:
I can see what you mean. The single value approach would make it clearer that we actually don't have Max,Mean,90p data that low. I'm +1 to doing that if you prefer.